CMSgov/mint-app
# Easi 2621/document solution link to user table

# EASI-2621

## Changes and Description

- Migrate document solution links to the user table. 



## How to test this change
1. Bring up the app.
2. Make sure Unit Tests pass.
3. Make sure you can add and remove links


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
